server/test/test_api.py

```diff
@@ -265,71 +265,81 @@ def test_filter_put_annotations_var(self):
         self.assertEqual(len(result_data["data"]), 2)
 
     def test_get_data(self):
-        endpoint = "data/obs"
-        query = "accept-type=application/json"
-        url = f"{URL_BASE}{endpoint}?{query}"
-        result = self.session.get(url)
-        self.assertEqual(result.status_code, 200)
-        result_data = result.json()
-        self.assertEqual(len(result_data["obs"]), 2638)
+        for axis in ["obs", "var"]:
+            endpoint = f"data/{axis}"
+            query = "accept-type=application/json"
+            url = f"{URL_BASE}{endpoint}?{query}"
```

Comment:
also need to test without a query param to validate that the default content-type is returned